scene/resources/compressed_texture.h

```diff
@@ -40,10 +40,11 @@ class CompressedTexture2D : public Texture2D {
 
 public:
 	enum DataFormat {
-		DATA_FORMAT_IMAGE,
-		DATA_FORMAT_PNG,
-		DATA_FORMAT_WEBP,
-		DATA_FORMAT_BASIS_UNIVERSAL,
+		DATA_FORMAT_UNDEFINED = 0,
```

Comment:
On the other hand it can cause confusion, and clutters

Reply:
Fine. Now I will remove these values and rebase my branch to the current master. But if the order of these names is changed in the future, this will break compatibility with already created resources.